hal/src/electron/modem/mdm_hal.cpp

```diff
@@ -418,34 +426,38 @@ int MDMParser::process() {
 
     {
         LOCK();
-        if (!_checkModem(false /* force */)) {
+        if (_reinitModem == 0x55 || !_checkModem(false /* force */)) {
+            if (_reinitModem != 0x55) {
+                MDM_ERROR("[ Modem not responsive ]\r\n");
```

Comment:
May be Nit: Looks like now `_checkModem()` failure will not have any log statement?.... Might be good to have similar log statement if `_checkModem()` fails also?

Reply:
_checkModem()'s log is "[ Modem not responsive ]", which is entry on != 0x55.  _reinitModem's logs happens in waitFinalResp().